components/Venue/venue.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import type { Meta, StoryObj } from '@storybook/nextjs';
+import React from 'react';
+
+import Venue from './venue';
+import { City } from '../../types/types';
+
+const meta: Meta<typeof Venue> = {
+  title: 'Components/Card/VenueCard',
+  component: Venue,
+  argTypes: {
+    city: {
+      control: { type: 'object' },
+      description: 'City object containing venue information',
+    },
+  },
+  decorators: [
+    (Story) => (
+      <div className="p-8">
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Venue>;
+
+const sampleCity: City = {
+  name: 'Singapore',
+  country: 'Singapore',
+  date: '9 - 11 December, 2025',
+  cfpDate: '5 October 2025',
+  description: "Let's talk event-driven architecture at the AsyncAPI Conference in Singapore! Join the open-source community and learn how to speak fluent API.",
+  img: '/img/locations/singapore.webp',
+  address: 'Marina Bay Sands, Singapore',
+  mapUrl: 'https://maps.app.goo.gl/9fG91E7Y24zmWSqP9',
+  sponsors: {
+    eventSponsors: [
+      {
+        image: '/img/logos/apidays.png',
+        websiteUrl: 'https://www.apidays.global/'
+      }
+    ]
+  },
+  freeEntry: true,
+  cfp: null,
+  recordings: null,
+  playlist: null
+};
+
+const pastEventCity: City = {
+  name: 'London',
+  country: 'UK',
+  date: '24 September, 2023',
+  cfpDate: '27 July 2023',
+  description: 'API enthusiasts in London! Join the AsyncAPI Conference for deep dives into event-driven architecture and open-source collaboration.',
+  img: '/img/locations/london.webp',
+  address: '155 Bishopsgate, London EC2M 3YD',
+  mapUrl: 'https://maps.app.goo.gl/b2Vb5H2mM41F9nQA6',
+  sponsors: {
+    eventSponsors: [
+      {
+        image: '/img/logos/apidays.png',
+        websiteUrl: 'https://www.apidays.global/'
+      }
+    ]
+  },
+  freeEntry: true,
+  cfp: null,
+  recordings: 'https://youtube.com/playlist?list=PLbi1gRlP7pijUwZJErzyYf_Rc-PWu4lXS',
+  playlist: 'https://youtube.com/playlist?list=PLbi1gRlP7pijUwZJErzyYf_Rc-PWu4lXS'
+};
```

Comment:
I think we should only have `upcoming` `past` and `cfp` card type... Also, for the upcoming, i'd advise you use a date sometime in the future, else the `ended` tag is gonna keep showing. Just as seen below. 
<img width="836" height="1028" alt="Components-Card-VenueCard-CFP-Open-⋅-Storybook-12-12-2025_07_11_PM" src="https://github.com/user-attachments/assets/7e24d431-c07e-4279-822b-f0e971840d61" />